src/scheduler.py

```diff
@@ -72,8 +72,12 @@ def _stop(self, sub_id):
         self._cleanup_paths()
 
     def _run_job(self, job_config, runtime, platform, input_node):
-        node = self._api_helper.create_job_node(job_config, input_node,
-                                                runtime, platform)
+        try:
+            node = self._api_helper.create_job_node(job_config, input_node,
+                                                    runtime, platform)
+        except Exception as e:
+            self.log.error(f"Error generating job node: {e}")
+            return
```

Comment:
The `try-catch` block is already implemented in the base class https://github.com/kernelci/kernelci-pipeline/blob/main/src/base.py#L72.